portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/customfields/cpcustomfields/CPCustomfields.testcase

```diff
@@ -204,6 +204,8 @@ definition {
 	@description = "This is a use case for LPS-146795. The user admin could add new user with space character in geolocation custom field name."
 	@priority = 4
 	test AddNewUserWithGeolocationCustomFields {
+		property portal.upstream = "quarantine";
+
```

Comment:
@joshitagaki This is under "Instance Setting" component, need to notify Platform Experience Team that you are adding this test to quarantine. Their team will need to modify their test suite to catch quarantine